cmd/dockerd/daemon_linux.go

```diff
@@ -16,3 +22,20 @@ func notifyReady() {
 func notifyStopping() {
 	go systemdDaemon.SdNotify(false, systemdDaemon.SdNotifyStopping)
 }
+
+func validateCPURealtimeOptions(config *config.Config) error {
+	if config.CPURealtimePeriod == 0 && config.CPURealtimeRuntime == 0 {
+		return nil
+	}
+
+	if cdcgroups.Mode() == cdcgroups.Unified {
+		return errors.New("daemon-scoped cpu-rt-period and cpu-rt-runtime are not implemented for cgroup v2")
+	}
+
+	// FIXME this is very expensive way to check if cpu rt is supported
```

Comment:
Is it expensive enough that we should skip it for now or just that we should think about maybe finding an alternative "eventually" ? (Given that this is a comment and the check is here, I'm assuming the latter? :see_no_evil:)

Reply:
Removed the TODO / comment 👍